README.md

```diff
@@ -119,6 +118,7 @@ of this library and directly using the API of the mysql library on which this on
 try {
   const connection = await mysql.getConnection();
   // Use the connection.
+  connection.release();
```

Comment:
Should this be done in a `finally` block?

Reply:
If the `getConnection` promise fails, I don't think there will be a connection to release.